jupyterhub/handlers/base.py

```diff
@@ -366,6 +366,18 @@ def spawner_class(self):
     def spawn_single_user(self, user, server_name='', options=None):
         if server_name in user.spawners and user.spawners[server_name]._spawn_pending:
             raise RuntimeError("Spawn already pending for: %s" % user.name)
+
+        if self.settings['concurrent_spawn_limit'] is not None and \
```

Comment:
Let's remove the `\` and put all in parens. [Ref](http://docs.python-guide.org/en/latest/writing/style/#line-continuations)